### PR TITLE
COM-343: Add campaigns admin

### PR DESCRIPTION
**Pull request by RainbowBunchie**
Add admin grid & form for email campaigns. 
I will add a section for test emails & a "send now" button in a follow up PR. 
The preview in the demo is not working since there is no frontend for the email campaigns as of now in the demo. 

Screenshot 2024-02-01 at 09 01 44
Screenshot 2024-02-01 at 09 02 05
Screenshot 2024-02-01 at 09 02 18
Screenshot 2024-02-01 at 09 02 43

This is part of COM-343

### CODE REVIEW

File: packages/api/src/email-campaign/dto/email-campaign-input.factory.ts [hunk 1]
```diff
@@ -27,13 +31,13 @@ export class EmailCampaignInputFactory {
             @Field()
             subject: string;
 
-            @IsUndefinable()
+            @IsNullable()
```
Comment (raphaelblum) [inline packages/api/src/email-campaign/dto/email-campaign-input.factory.ts:34]:
```suggestion
      @IsUndefinable()
```

The type is not Date | null

File: packages/api/src/email-campaign/dto/email-campaign-input.factory.ts [hunk 2]
```diff
@@ -27,13 +31,13 @@ export class EmailCampaignInputFactory {
...
             @IsDate()
             @MinDate(new Date())
             @Field(() => Date, { nullable: true })
             scheduledAt?: Date;
 
-            @IsUndefinable()
+            @IsNullable()
```
Comment (raphaelblum) [inline packages/api/src/email-campaign/dto/email-campaign-input.factory.ts:40]:
```suggestion
            @ IsUndefinable()
```

The type is not Date | null

Comment (RainbowBunchie) [reply packages/api/src/email-campaign/dto/email-campaign-input.factory.ts]: Adapted in [9ff247e](https://github.com/vivid-planet/comet-brevo-module/pull/32/commits/9ff247ecf83b670fe2fc5ee37a2b6ffbc63a50d8)

File: packages/api/src/email-campaign/dto/email-campaign-input.factory.ts (hunk 1 above)
Comment (RainbowBunchie) [reply packages/api/src/email-campaign/dto/email-campaign-input.factory.ts]: Adapted in [9ff247e](https://github.com/vivid-planet/comet-brevo-module/pull/32/commits/9ff247ecf83b670fe2fc5ee37a2b6ffbc63a50d8)

**Review by raphaelblum — APPROVED** [review]

**Pull request merged**